nncf/quantization/algo.py

```diff
@@ -693,9 +693,14 @@ def _add_single_weight_quantizer(self, target_model: NNCFNetwork, insertion_poin
         module = target_model.get_module_by_scope(insertion_point.module_scope)
         scale_shape = get_scale_shape(module.weight.shape, is_weights=True, per_channel=qconfig.per_channel)
         use_logarithm_scale = self._use_logarithm_scale_per_group[QuantizerGroup.WEIGHTS]
+        apply_saturation_fix = False
+        if self.hw_config:
+            if self.hw_config.target_device in ['CPU', 'ANY'] and qconfig.num_bits == 8:
+                apply_saturation_fix = True
```

Comment:
What do you think about printing a message that apply_saturation_fix mode is used? It can be not clear for new NNCF users.

Reply:
good idea. I will add

Reply:
Done